templates/idx.html

```diff
@@ -229,10 +226,10 @@
             messageDiv.innerHTML = messageHTML.join("<br />")
         }
 
-        $(".navbar-burger").click(function () {
-            $(".navbar-burger").toggleClass("is-active");
-            $(".navbar-menu").toggleClass("is-active");
-        });
+        // $(".navbar-burger").click(function () {
```

Comment:
Do not leave commented-out code, either remove it completely or add a `TODO` with an issue reference to re-enable it